packages/ui/src/components/input/tags/index.tsx

```diff
@@ -1,13 +1,22 @@
 import { cva } from "class-variance-authority";
 import React, { forwardRef, useCallback, useState } from "react";
 import { ReactElement } from "react";
+import { Button } from "../button";
 import { BaseInputProps, BaseInputWrapper, inputStyles } from "../commons";
 import { Tag, TagProps } from "./tag";
 
-const tagsWrapper = cva(["cui-flex", "cui-flex-wrap", "cui-gap-2", "cui-mt-2"]);
+const tagsWrapperStyles = cva([
+    "cui-flex",
+    "cui-flex-wrap",
+    "cui-gap-2",
+    "cui-mt-2",
+]);
+
+const inputWrapperStyles = cva(["cui-flex", "cui-gap-2"]);
 
 export type TagsInputProps = Omit<BaseInputProps<string[]>, "onChange"> & {
     onChange: (tags: string[]) => void;
+    addButtonLabel: string;
```

Comment:
The practice as of now has been to add a `message` object to components that need labels passed in.